packages/sqrl-cli/src/SqrlServer.ts

```diff
@@ -115,6 +115,7 @@ async function deleteRoute(
 
 export function createSqrlServer(ctx: Context, executable: Executable): Server {
   const router = dispatch()
+    .dispatch('/health', '*', (req, res, { params, query }) => send(res, 200, "Hello"))
```

Comment:
I think it's better to be conservative and only allow this on `"GET"`. May be a reason we want to enable `POST` later with a specific schema and not break anyones existing healthchecks.

Responding with `"Hello"` feels a little odd to me as well, something indicating it's a response from a healthcheck would be more useful (`Okay" / "Healthcheck Okay" / whatever).

Reply:
Sure. I can make those changes and update the PR